Assets/FullSerializer/Source/fsConfig.cs

```diff
@@ -53,7 +53,12 @@ public class fsConfig {
         public Func<string, MemberInfo, string> GetJsonNameFromMemberName = (name, info) => name;
 
         /// <summary>
-        /// Should the default serialization behaviour include non-auto properties?
+        /// Should the default serialization behaviour include any properties?
+        /// </summary>
+        public bool SerializeProperties = true;
```

Comment:
It's confusing having two different options for property serialization. How about introducing an enum, `PropertySerializationConfig`, that looks like:

```c#
enum PropertySerializationConfig {
    Disabled,
    OptInOnly,
    AutoProperties,
}
```

If you can think of better names please substitute them in.